### ISSUE TITLE
[MNG-7461] Maven to exclude list/wildcard of groupId's or artifactId's from particular repository

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#9135]
**[Kapil Singal](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=JIRAUSER288026)** opened **[MNG-7461](https://issues.apache.org/jira/browse/MNG-7461?redirect=false)** and commented

Let's say an artifact with GAV :: com.abc : xyz : 0.0.1-SNAOPSHOT is present in repositoryA and repositoryB.

Want Maven settings to define in such a way, so-that maven only try to pull artifact xyz from repositoryA.
In other words, wants repositoryA to be used only to pull xyz artifact or any artifact from com.abc group.

To achieve same, an includes/excludes xml chunk can be defined under repository definition for repositoryB within settings.xml

---

**Issue Links:**
- [MNG-6763](https://issues.apache.org/jira/browse/MNG-6763) Restrict repositories to specific groupIds
 (_**"duplicates"**_)
